.travis.yml

```diff
@@ -6,6 +6,9 @@ env:
   - TOXENV=django19
 
 install:
+  - nvm install
+  - npm install
```

Comment:
As a thought, given that [TravisCI supports Yarn natively](https://yarnpkg.com/en/docs/install-ci#travis-tab), we should consider configuring it to do so.